providers/common/include/prov/ciphercommon.h

```diff
@@ -233,6 +234,16 @@ static int cipher_hw_##NAME##_##MODE##_cipher(PROV_CIPHER_CTX *ctx,            \
     return 1;                                                                  \
 }
 
+#define IMPLEMENT_CIPHER_HW_COPYCTX(name, CTX_TYPE)                            \
+static void name (PROV_CIPHER_CTX *dst, const PROV_CIPHER_CTX *src)            \
```

Comment:
Space after `name` here

Reply:
NIT fixed.. I did that so I didnt have to figure out if ## was required (it isnt required :))